omp/btx_ompmatching_model.yaml

```diff
@@ -0,0 +1,15 @@
+:environment:
+  :entries:
+  - :name: ^hostname$
+    :type: string
+:stream_classes:
+- :default_clock_class: {}
+  :event_common_context_field_class:
+    :type: structure
+    :members:
+    - :name: ^vpid$
+      :field_class:
+        :type: integer_signed
+    - :name: ^vtid$
+      :field_class:
+        :type: integer_unsigned
```

Comment:
I don't see we need a matching model for OpenMP

Reply:
I was keeping it around just in case it became useful, but I can remove now, definitely not needed for parity with erb version.